ocs_ci/ocs/ui/views.py

```diff
@@ -1369,12 +1369,17 @@
         By.XPATH,
     ),
     "odf-performanceCardLink": (
-        "td[class='pf-u-w-10 performanceCard--verticalAlign'] a",
+        "td[class='pf-u-w-10 performanceCard--verticalAlign'] a, "
+        "td[class='pf-v5-c-table__td pf-v5-u-w-10 performanceCard--verticalAlign'] a",
```

Comment:
CSS_Selector is too long, is there any other locator available which is short but robust?

Reply:
Actually you're right, I can just replace it with class*='performanceCard--verticalAlign'. Thank you. Fixed

Reply:
this is **one class name** `pf-v5-c-table__td pf-v5-u-w-10 performanceCard--verticalAlign` as far as I see, so it will not make difference if we use long class name or short class name